### PR TITLE
Secondary branch

### PR DESCRIPTION
**Pull request by pountzas**
# Added files
- index.html 
- styles.css
- /images/

### CODE REVIEW

File: index.html
Comment (GzimAsani) [inline index.html]: Please change the color of these buttons and make them look as the original

File: index.html
Comment (GzimAsani) [inline index.html:51]: This line is extending more than the picture, kindly fix it

File: index.html
Comment (GzimAsani) [inline index.html:73]: This section is extending more than it should in to the right side which is making your site to expand more than it should, kindly fix this.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:
Hello **Team** 👋 Good job implementing so far :muscle:, however there are some more changes that you have to implement

Required changes:
- [ ] Add a more detailed description in your pull request
- [ ] Add social media icons which are between title and the main picture 
- [ ] Add the comments button and social media icons at the bottom of the page, this should be after  the line div in line 205
- [ ] Add the `More in Space and Astronomy` and `Most Popular` sections 
- [ ] You are using repated css code like _display: flex_ 4 times and float 5 times this is bad CSS practice what you can do is create a class like `.grid { display: grid; }` and use it every time you need
- [ ] Kindly check the comments below

# You got this :muscle: 

### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

File: index.html
Comment (umairarshadbutt) [inline index.html]: Remove inline Styling

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi Team
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested # changes 👇. 
- [ ] Add a more detailed description in your pull request.
- [ ] Add the More in Space and Astronomy and Most Popular sections
- [ ] Remove inline Styling
- [ ] Please add MIT license in the README file

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @pountzas :wave:

__You've done excellently well so far. Your project is looking ready for approval :wink: 
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] You've highlighted how to clone the project in the Setup section of the Readme, please complete this section by pointing out how to run the project on the browser to your users. It is always better to have a complete Readme without any assumption that the users know how to find their way.
- [ ] The `Issues page` and the `MIT license` links provided on the readme are broken. Please ensure that the MIT license file is present and the links are working.
- [x] Kindly check the image below. I believe the date is in the wrong place.
date

- [x] Your `Most Popular` section is a bit different from the one on the original page. Please check the table below and make necessary adjustments: take note of the spacings and alignments.

| Your Page | Original Page |
| -------------- | ------------------ |
| clone | original |

**[ OPTIONAL CHANGES ]**

- [x] For subsequent commits, try to be more professional with your commit messages. I believe using messages like `code review fix 7th try test1` is not the best. Check out [this article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) on how to write better commit messages.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**
